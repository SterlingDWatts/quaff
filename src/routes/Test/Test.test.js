import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
import Test from "./Test";

describe("Test Component", () => {
  it("render without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <Test />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders the UI as expected", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Test />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
