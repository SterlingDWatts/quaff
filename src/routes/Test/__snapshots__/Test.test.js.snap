// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Component renders the UI as expected 1`] = `
<div
  className="Test"
>
  <div
    className="QuaffSquare "
  >
    <div
      className="QuaffSquare__container"
    >
      <div
        className="Test--loading"
      />
    </div>
  </div>
  <div
    className="NextUp"
  >
    <div
      className="NextUp__learn-loading"
    >
      <div
        className="NextUp__header"
      >
        Learn
      </div>
      <div
        className="NextUp__container"
      >
        Loading
      </div>
    </div>
    <div
      className="NextUp__study-loading"
    >
      <div
        className="NextUp__header"
      >
        Learn
      </div>
      <div
        className="NextUp__container"
      >
        Loading
      </div>
    </div>
    <div
      className="NextUp__study-loading"
    >
      <div
        className="NextUp__header"
      >
        Learn
      </div>
      <div
        className="NextUp__container"
      >
        Loading
      </div>
    </div>
    <div
      className="NextUp__study-loading"
    >
      <div
        className="NextUp__header"
      >
        Learn
      </div>
      <div
        className="NextUp__container"
      >
        Loading
      </div>
    </div>
  </div>
</div>
`;
